qa/workunits/suites/mgr_mod_snap_schedule.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/bash
+TESTDIR=$(dirname $0)
+
+echo "TESTDIR:$TESTDIR"
+
+ceph mgr module enable snap_schedule
+sleep 2
+ceph config set mgr mgr/snap_schedule/allow_m_granularity True
+sleep 2
+mkdir $CEPH_MNT/t
+ceph fs snap-schedule add /t 1M
+pushd $CEPH_MNT/t
+
+(python -c "import os
+from datetime import time
+
+fd = os.open('./data-file', os.O_WRONLY | os.O_CREAT, 0o777)
+assert fd is not None'
```

Comment:
hmm; the script may not fail since the snapshot creation may have succeeded
the background script is just an I/O generator
I think it will be appropriate to fail the script if the background I/O generation fails ?

Reply:
right!